README.md

```diff
@@ -97,6 +97,35 @@ original module. This code depends on the following maven package (should be def
 
 The output directory will also include a tarball `generated@0.0.0.jsii.tgz` that must be bundled in your project.
 
+### .NET Output
+
+To produce a .NET module from your source, use the `dotnet` option:
+
+```ts
+await srcmak('srcdir', {
+  dotnet: {
```

Comment:
I'd go with `csharp` here since this is producing source code and not a library (like Pacmak)